src/browser/atom-window.coffee

```diff
@@ -73,6 +73,9 @@ class AtomWindow
     @browserWindow.loadUrl @getUrl(loadSettings)
     @browserWindow.focusOnWebView() if @isSpec
 
+    if locationsToOpen.length is 1 and locationsToOpen[0].pathToOpen is null
```

Comment:
Can you do this earlier? At what point do we decide to open the initial empty buffer? Seems like you could do it then.